sandy/decay.py

```diff
@@ -973,6 +882,353 @@ def to_hdf5(self, filename, lib, mode="a"):
                         data,
                         )
 
+class _DecayBase():
+    """
+    Base class to perturb decay data
+
+    Attributes
+    ----------
+    data
+        best estimates and uncertainty or only best estimates as a dataframe
+
+    Methods
+    -------
+    custom_perturbation
+        apply custom perturbation to a given `BranchingRatio`, `DecayEnergy` 
+        or `HalfLife` instance.
+    """
+
+    def __init__(self, df):
+        self.data = pd.DataFrame(df)
+
+    def __repr__(self):
+        return self.data.__repr__()
+
+    def custom_perturbation(self, pert):
+        """
+        Apply a custom perturbation to a given `BranchingRatio`, `DecayEnergy` 
+        or `HalfLife` instance.
+
+        Parameters
+        ----------
+        pert : `pandas.DataFrame`
+            dataframe containing perturbation coefficients as ratio values,
+            e.g., 1.05 for a perturbation of +5%.
+            Depending on the nuclear data to perturb, `pert` index should be:
+                * if perturbing branching ratio: "ZAM", "RTYP"
+                * if perturbing decay energy: "ZAM", "TYPE"
+                * if perturbing half life: "ZAM"
```

Comment:
Do I have to add also the descriptions of "ZAM", "RTYP" and "TYPE"? I think that I have to specify that the RTYP must be a string and it is different from the RTYP value found in ENDF6 format, should it be a note? 

P.s. I'm checking the test for which it fails

Reply:
Should be ok as it is